src/blackchocobo.cpp

```diff
@@ -150,14 +156,18 @@ void BlackChocobo::detectTranslations()
             }
         }
     }
+    QRegularExpression qtRe("^qt_([a-z]{2}(?:_[A-Z]{2})?)\\.qm$", QRegularExpression::CaseInsensitiveOption);
     for (const QString &translation : std::as_const(langList)) {
         QTranslator *translator = new QTranslator;
         std::ignore = translator->load(translation, dir.absolutePath());
-        QString lang = translation.mid(3, 2);
-        qt_translations.insert(lang, translator);
-        bool currentLang = (BCSettings::value(SETTINGS::LANG, QStringLiteral("en")).toString() == lang);
-        if (currentLang)
-            QApplication::installTranslator(translator);
+        QRegularExpressionMatch match = qtRe.match(translation);
```

Comment:
This does not seam to correctly load the Qt translations. When they are loaded correctly the labels on the settings window should be show in the same language as the app (chinese in this case)